index.html

```diff
@@ -187,12 +185,18 @@ <h2 class="footer-headline">I'm always interested in hearing about new projects,
       <textarea class="footer-input" name="text-area" id="text-area" placeholder="Enter text here" rows="5" maxlength="500" required></textarea>
       <button class="footer-button see-project" type="submit">Get in touch</button>
```

Comment:
- [ ] The default and active states for your buttons are great 👏. Please reference the image below and use it to implement correctly the hover state for the buttons 🙏 
![image](https://user-images.githubusercontent.com/46839548/166037730-61ca6dad-29d7-4a10-8e3a-025cfb560c72.png)

Reply:
done, thanks